src/main/java/co/omise/models/Charge.java

```diff
@@ -40,6 +40,7 @@
     private OffsiteTypes offsite;
```

Comment:
Since we have the `Source` type to describe this. Should we move this `OffsiteTypes` into the Source?